src/components/OSCALCatalogGroups.test.js

```diff
@@ -35,6 +35,22 @@ const testGroups = [
   },
 ];
 
+function getTextByChildren(text) {
```

Comment:
This is necessary because we're now potentially pulling text across multiple `<span>`s.